packages/nuxt/src/module.ts

```diff
@@ -63,22 +64,29 @@ export default defineNuxtModule<ModuleOptions>({
     if (clientConfigFile || serverConfigFile) {
       setupSourceMaps(moduleOptions, nuxt);
     }
+    nuxt.hooks.hook('nitro:init', nitro => {
+      if (serverConfigFile && serverConfigFile.includes('.server.config')) {
+        addServerConfigToBuild(moduleOptions, nuxt, serverConfigFile);
 
-    if (serverConfigFile && serverConfigFile.includes('.server.config')) {
-      addServerConfigToBuild(moduleOptions, nuxt, serverConfigFile);
+        if (moduleOptions.experimental_basicServerTracing) {
+          addSentryTopImport(moduleOptions, nuxt);
+        } else {
+          if (moduleOptions.debug) {
+            const serverDirResolver = createResolver(nitro.options.output.serverDir);
+            const serverConfigPath = serverDirResolver.resolve('sentry.server.config.mjs');
 
-      if (moduleOptions.experimental_basicServerTracing) {
-        addSentryTopImport(moduleOptions, nuxt);
-      } else {
-        if (moduleOptions.debug) {
-          consoleSandbox(() => {
-            // eslint-disable-next-line no-console
-            console.log(
-              `[Sentry] Using your \`${serverConfigFile}\` file for the server-side Sentry configuration. In case you have a \`public/instrument.server\` file, the \`public/instrument.server\` file will be ignored. Make sure the file path in your node \`--import\` option matches the Sentry server config file in your \`.output\` folder and has a \`.mjs\` extension.`,
-            );
-          });
+            // For the default nitro node-preset build output this relative path would be: ./.output/server/sentry.server.config.mjs
+            const serverConfigRelativePath = `./${path.relative(nitro.options.rootDir, serverConfigPath)}`;
```

Comment:
Let's use `path.join` instead of `./` as the `/` separator does not work in windows. In this case doesn't matter a ton, but always good to it consistent.

Reply:
Good catch with windows! I could not use `path.join` in this case as this would not keep the folder name with the dot (`.output`). But I was able to achieve this with `.${path.sep}${path.relative(...)}`